gtfs.yml

```diff
@@ -290,6 +290,156 @@
       text: Gondola
     - value: 7
       text: Funicular
```

Comment:
Could you also add options 11 and 12 (they were recently added to the standard set)? See the spec for `routes.txt`: https://developers.google.com/transit/gtfs/reference#routestxt for more info

Reply:
Done!